earth_enterprise/rpms/opengee-fusion/post-install.sh

```diff
@@ -148,11 +157,7 @@ install_or_upgrade_asset_root()
     else
         # upgrade asset root -- if this is a master
         if [ "$IS_SLAVE" = false ]; then
-            # TODO: Verify this logic -- this is what is defined in the
-            # installer documentation, but needs confirmation
-            keyvalue_file_get "$GEE_INSTALL_KV_PATH" gegroup_existed NEW_GEFUSIONUSER
-            keyvalue_file_get "$GEE_INSTALL_KV_PATH" gefusionuser_existed NEW_GEFUSIONUSER
-            if [ "$NEW_GEGROUP" = true ] || [ "$NEW_GEFUSIONUSER" = true ]; then
+            if [ "$NEW_INSTALL" = false ] ; then
```

Comment:
Q: How does this test if installing the `opengee-common` package created a new `gegroup` or not?

Q: Can we test whether to 'upgrade' by checking if one of the top-level directories has the wrong permissions?  Do we know upgrading from what version of GEE to what version of GEE requires changing file ownership?

Reply:
We now know that opengee-common creates in it's pre-install now, and does not remove it all (or only in a post-uninstall when not upgrading), so we assume it's there based on this.  There are other/other kinds of tests we could do, but I was trying to keep consistent with practices common to other packages.

Reply:
There are things we also probably still over do, and could do more checks for, but I did not want to invalidate all prior testing, and so focused on the issues around package updating itself only rather than about some of the extra and misc warning messages we still do get.